### ISSUE TITLE
Final Forecast Issues

### ISSUE THREAD

**Issue by samuelburge** [maxwkut/Web-Traffic-Forecasting#5]
There's a few issues with the final forecast that I'm having some trouble with:

- [ ] Should we just use the model we fit using the training data to forecast an additional 30 days beyond what we already forecasted for computing the accuracy measures/generalization error from the test set? Or just refit an ARMA(p,q) model using the full dataset and then forecasting 30 days out using that? Not sure what the best approach is on that.
- [x] (2.) Regardless, I'm having some issues with computing the back-transformations on the 30 days because of the weekly lag. Since the forecast goes beyond the available information, I'll have to start using the one-step-ahead predictions for as the unique visits, and I'm not sure what the best way to do that using the `tidyverse` packages are. I think I'll probably just have to run a loop, but that's incomplete in the file I just uploaded.

**Comment by maxwkut** [comment]
1) I would compute the test error on the test set and then refit the best model on the entire dataset to make a forecast. 

2) We could just do 7-day forecasts to make it easier

**Comment by samuelburge** [comment]
Okay. I would agree with you on fitting the best model to the *full* data set, so I've done that. I also got the issue with the one-step-ahead prediction squared away using that loop you had tried using for the back-transformations previously (that worked like a charm). I have pushed out an updated script that we can start parsing through and working more on the write-up.
